File: SSNodeJsChat-Android/app/src/main/java/in/newdevpoint/ssnodejschat/model/UserBlockModel.java
package in.newdevpoint.ssnodejschat.model;

import com.google.gson.annotations.SerializedName;

import java.io.Serializable;

public class UserBlockModel implements Serializable {

    @SerializedName("blockedBy")
    private String blockedBy = "";


    @SerializedName("blockedTo")
    private String blockedTo = "";


    @SerializedName("isBlock")
    private boolean isBlock = false;


    public String getBlockedBy() {
        return blockedBy;
    }

    public void setBlockedBy(String blockedBy) {
        this.blockedBy = blockedBy;
    }

    public String getBlockedTo() {
        return blockedTo;
    }

    public void setBlockedTo(String blockedTo) {
        this.blockedTo = blockedTo;
    }

    public boolean isBlock() {
        return isBlock;
    }

    public void setBlock(boolean block) {
        isBlock = block;
    }

    public UserBlockModel() {
    }

}

File: SSNodeJsChat-Android-Kotlin/app/src/main/java/in/newdevpoint/ssnodejschat/model/UserBlockModel.kt
package `in`.newdevpoint.ssnodejschat.model

import com.google.gson.annotations.SerializedName
import java.io.Serializable

class UserBlockModel : Serializable {
    @SerializedName("blockedBy")
    var blockedBy = ""

    @SerializedName("blockedTo")
    var blockedTo = ""

    @SerializedName("isBlock")
    var isBlock = false
}